daemon/api.go

```diff
@@ -2627,13 +2627,17 @@ func postApps(c *Command, r *http.Request, user *auth.UserState) Response {
 		return InternalError("no services found")
 	}
 
-	chg, err := servicestate.Change(st, appInfos, &inst)
+	ts, err := servicestate.Change(st, appInfos, &inst)
 	if err != nil {
 		if _, ok := err.(servicestate.ServiceActionConflictError); ok {
 			return Conflict(err.Error())
 		}
 		return BadRequest(err.Error())
 	}
+	st.Lock()
+	defer st.Unlock()
+	chg := st.NewChange("service-control", fmt.Sprintf("Running service command"))
```

Comment:
It looks like `newChange(st, "service-control", fmt.Sprintf("Running service command"), []*state.TaskSet{ts}, inst.Names)` is what we do for most new changes (except in the changeAlias which looks like it might be an oversight?).

Reply:
Thanks for pointing that out, done.